packages/server/src/db/utils.ts

```diff
@@ -284,10 +284,20 @@ export function getMultiIDParams(ids: string[]) {
  * @returns {string} The new view ID which the view doc can be stored under.
  */
 export function generateViewID(tableId: string) {
-  return `${tableId}${SEPARATOR}${newid()}`
+  return `${DocumentType.VIEW}${SEPARATOR}${tableId}${SEPARATOR}${newid()}`
```

Comment:
This id will not be part of a document. Maybe this shouldn't be a `documenttype`

Reply:
I'll add a `VirtualDocumentType` that might be less confusing